gnocchi/tests/test_carbonara.py

```diff
@@ -19,13 +19,17 @@
 
 import fixtures
 import iso8601
-import pandas
+import numpy
 import six
 
 from gnocchi import carbonara
 from gnocchi.tests import base
 
 
+def datetime64(*args):
```

Comment:
for the non-assert stuff i mean... i guess it doesn't matter... it's a sh*tload of code regardless